src/components/People.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import PeopleContext from '../contexts/PeopleContext';
+
+export const People = React.memo(() => {
+  const {
+    input,
+    setInput,
+    applyQueryWithDebounce,
+    filteredPeople,
+    handleClick,
+  } = useContext(PeopleContext);
+
+  const handleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const userInput = event.target.value;
+
+    applyQueryWithDebounce(userInput);
+
+    setInput(userInput);
+  };
+
+  return (
+    <div className="dropdown is-active">
+      <div className="dropdown-trigger">
+        <input
+          type="search"
+          placeholder="Enter the name"
+          className="input"
+          value={input}
+          onChange={handleInput}
+        />
+      </div>
+
+      {input && (
+        <div className="dropdown-menu" role="menu">
+          <div className="dropdown-content">
+            {filteredPeople.length > 0
```

Comment:
Could be just `filteredPeople.length` instead of `filteredPeople.length > 0`[as `0` is `false`](https://www.freecodecamp.org/news/falsy-values-in-javascript/)